sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -877,6 +912,26 @@ private void setDefaultTransmissionTarget(String transmissionTargetToken) {
         }
     }
 
+    /**
+     * Set transmission interval. The interval should be between 3 seconds and 1 day.
+     * Should be called before the service is started.
+     *
+     * @param seconds Set latency of sending events to Analytics.
+     * @return <code>true</code> if the interval is set, <code>false</code> otherwise.
+     */
+    private boolean setInstanceTransmissionInterval(int seconds) {
```

Comment:
Will it be remaining as an int value?

Reply:
its passed in seconds, stored in milliseconds.

Reply:
Any reason not to use `long` type?

Reply:
We pass in int seconds, not need for long given the allowed values.
Long is used for internals of the SDK that uses milliseconds.